pkg/pillar/nireconciler/linux_config.go

```diff
@@ -810,17 +842,64 @@ func (r *LinuxNIReconciler) getIntendedDnsmasqCfg(niID uuid.UUID) (items []dg.It
 		ntpServers = append(ntpServers, ni.config.NtpServer)
 	}
 	ntpServers = generics.FilterDuplicatesFn(ntpServers, netutils.EqualIPs)
+	var propagateRoutes []types.IPRoute
```

Comment:
Why do you do this work even if PropagateConnRoutes is not set?

Reply:
This list is also used for static IP routes, which are always propagated (see [here](https://github.com/lf-edge/eve-api/blob/main/proto/config/netinst.proto#L139-L142)).

Reply:
Ok